### PR TITLE
Implement B006

### PR DESCRIPTION
**Pull request by harupy**
See #389

### CODE REVIEW

File: src/checks_gen.rs [hunk 1]
```diff
@@ -887,6 +891,7 @@ impl CheckCodePrefix {
             CheckCodePrefix::B0 => PrefixSpecificity::Hundreds,
             CheckCodePrefix::B00 => PrefixSpecificity::Tens,
             CheckCodePrefix::B002 => PrefixSpecificity::Explicit,
+            CheckCodePrefix::B006 => PrefixSpecificity::Explicit,
```
Comment (charliermarsh) [inline src/checks_gen.rs:894]: @harupy - BTW -- any thoughts on the approach I took to this problem? I.e., code-generating the prefix map, to allow static enums everywhere?

File: resources/test/fixtures/B006.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import collections
```
Comment (charliermarsh) [inline resources/test/fixtures/B006.py:1]: Could we instead use the test suite from flake8-bugbear? https://github.com/PyCQA/flake8-bugbear/blob/main/tests/b006_b008.py

File: src/flake8_bugbear/plugins/mutable_argument_default.rs [hunk 3]
```diff
@@ -0,0 +1,62 @@
+use rustpython_ast::{Arguments, ExprKind};
+
+use crate::ast::types::{CheckLocator, Range};
+use crate::check_ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+/// B006
+pub fn mutable_argument_default(checker: &mut Checker, arguments: &Arguments) {
+    for default in arguments
```
Comment (charliermarsh) [inline src/flake8_bugbear/plugins/mutable_argument_default.rs:9]: Nit: can we rename `default` to `arg` or `expr`? `default` is a keyword so the syntax highlighting is confusing here.

**Comment by charliermarsh** [comment]
Awesome, a couple comments!

File: src/flake8_bugbear/plugins/mutable_argument_default.rs (hunk 3 above)
Comment (harupy) [reply src/flake8_bugbear/plugins/mutable_argument_default.rs:9]: Got it, will change this.

Comment (harupy) [reply src/flake8_bugbear/plugins/mutable_argument_default.rs:9]: How about `deafult_value`?

Comment (harupy) [reply src/flake8_bugbear/plugins/mutable_argument_default.rs:9]: Chose `expr`.

File: src/checks_gen.rs (hunk 1 above)
Comment (harupy) [reply src/checks_gen.rs:894]: How does flake8 implement this?

**Pull request merged**